index.js

```diff
@@ -1,16 +1,10 @@
 /*! node-domexception. MIT License. Jimmy Wärting <https://jimmy.warting.se/opensource> */
 
-if (!globalThis.DOMException) {
-  try {
-    const { MessageChannel } = require('worker_threads'),
-    port = new MessageChannel().port1,
-    ab = new ArrayBuffer()
-    port.postMessage(ab, [ab, ab])
-  } catch (err) {
-    err.constructor.name === 'DOMException' && (
-      globalThis.DOMException = err.constructor
-    )
-  }
-}
+const DOMException = globalThis.DOMException ??= (() => {
+  try { atob(0) } catch (err) { return err.constructor }
+})()
 
-module.exports = globalThis.DOMException
+export {
```

Comment:
why the esm export requirement, this seems unnecessary

Reply:
Do you think it would be better to just ditch `export` entirely so that it's only a pollyfill? 
essentially also removing the `"type": "module"` thing in pkg json also. so it can work with both cjs and esm?

Reply:
yeah.

Reply:
Okey then.